File: src/main/java/no/nav/pensjon/regler/domain/beregning2011/Fremskriving.java
package no.nav.pensjon.regler.domain.beregning2011;

import no.nav.pensjon.regler.domain.kode.FremskrevetMPNTypeCti;

public interface Fremskriving {

    public void setTeller(int teller);

    public int getTeller();

    public void setNevner(int nevner);

    public int getNevner();

    public void setBrok(double brok);

    public double getBrok();

    public void setType(FremskrevetMPNTypeCti type);

    public FremskrevetMPNTypeCti getType();
}

File: src/main/kotlin/no/nav/pensjon/regler/domain/beregning2011/Fremskriving.kt
package no.nav.pensjon.regler.domain.beregning2011

import no.nav.pensjon.regler.domain.kode.FremskrevetMPNTypeCti

@Deprecated("Ikke i bruk")
interface Fremskriving {
    var teller: Int
    var nevner: Int
    var brok: Double
    var type: FremskrevetMPNTypeCti?
    // TODO Hva skal brukes her?
    var typeEnum: FremskrevetMPNTypeCti?
}